app/assess/helpers.py

```diff
@@ -127,3 +128,52 @@ def generate_text_of_application(q_and_a: dict, fund_name: str):
             output.write(f"  Q) {questions}\n")
             output.write(f"  A) {answers}\n\n")
     return output.getvalue()
+
+
+def replace_none_location(
```

Comment:
Yes, we don't for a prod I guess but for the local environment in case the incomplete data is pulled in for some reason so we wanna avoid UI breaking.

Reply:
I think with the updates you made in assessment-store (https://github.com/communitiesuk/funding-service-design-assessment-store/pull/176). Now we handle the location data in better way when applications are imported from application_store both for partially completed applications & missing or invalid location information applications. I think it's already fixed in assessment store. Can we take others opinions if these changes are needed here in assessment ? @Adam-W1

Reply:
This function 👆 (https://github.com/communitiesuk/funding-service-design-assessment-store/pull/176) would not return the default data if there is no postcode etc however we do have a fix here (https://github.com/communitiesuk/funding-service-design-assessment-store/pull/176) which returns the default data if there was no location address. 
I do agree with you though but yea let's take some other opinion. I had already asked Adam W in the morning so waiting on him to review it.

Reply:
I also think it's a little unnecessary, and not too big of a fan of using side-effects in this case:

I think something like this would be better:
https://github.com/communitiesuk/funding-service-design-assessment/blob/e913bee572044b6c7ef9d8654093b23035b6ab6d/app/assess/templates/components/application_overviews_table.html#L137

Less verbose and handled on the view, although if using this value elsewhere we'd need to duplicate the logic, but there is only one instance of it.

Reply:
Agree! removed! 😄

Reply:
👍 LGTM, lets see what @RamuniN thinks in the morning